src/backupNg/backupNg.spec.js

```diff
@@ -60,14 +59,13 @@ describe("backupNg", () => {
       });
       expect(backupNgJob.userId).toBe(xo._user.id);
 
-      const settingKeys = Object.keys(backupNgJob.settings);
-      expect(settingKeys.length).toBe(2);
-      const scheduleId = settingKeys.find(key => key !== "");
-      expect(backupNgJob.settings[scheduleId]).toEqual({
+      expect(Object.keys(backupNgJob.settings).length).toBe(2);
+      const schedule = await xo.getSchedule({ jobId });
```

Comment:
[This test is not anymore relevant](https://github.com/vatesfr/xo-test/pull/99/files#diff-082da5cf9ebc8256029b93414f29ac5fR73)